tests/iface_namingmode/test_iface_namingmode.py

```diff
@@ -120,6 +120,9 @@ def setup_config_mode(ansible_adhoc, duthosts, enum_rand_one_per_hwsku_frontend_
     logger.info('Creating a guest user')
     duthost.user(name='guest', groups='sudo', state ='present', shell='/bin/bash')
     duthost.shell('echo guest:guest | sudo chpasswd')
+    # Add the guest user to the redis group such that it can execute show commands that need access to the redis database
+    # One such command is 'show acl table'
+    duthost.shell('sudo usermod -aG redis guest')
```

Comment:
Why is this needed now and was not previously required?
Seems like this works fine on single asic but not on multi-asic.

Reply:
show acl table command should be allowed for read-only user, this should be fixed in acl-loader.
@arlakshm fyi

Reply:
Assuming acl-loader is in sonic-buildimage. Is the fix already merged. If not,  is there an ETA on it.

Reply:
Fix is not merged in sonic-buildimage, PR will be available from @arlakshm soon.

Reply:
@sanmalho-git This change can be removed from here to add guest uesr to redis group.
This is the PR to ensure the cli will work for RO user: https://github.com/sonic-net/sonic-utilities/pull/2525

Reply:
We could remove this change from this PR and merge this PR.

We will track sonic-net/sonic-utilities#2525 this PR to handle the permission error that we see with show acl table.

Reply:
@SuvarnaMeenakshi  - I have removed the fix in iface_namingmode.py that will be fixed by https://github.com/sonic-net/sonic-utilities/pull/2525